modules/library.js

```diff
@@ -51,4 +51,4 @@ export default class Library {
       localStorage.setItem('books', JSON.stringify(this.books));
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
- [ ] Kindly change this function `function storageAvailable(type)` to an ES6 arrow function. hint;

```js
const storageAvailable = (type) => {...}

Reply:
Function updated 👍🏽